docs/documentation/updating-the-kit.md

```diff
@@ -13,43 +13,39 @@ Tell us as much as you can about the issue you're having, and the computer and o
 
 ## Updating to the latest version
 
-1. [Download the zip file of the latest version of the Prototype Kit](/docs/download).
+### Find what version you're using
 
-2. Unzip the zip file. It will make a folder called `govuk-prototype-kit`, with a version number.
+In Finder on Mac or Windows Explorer go to your prototype folder and open the file `VERSION.txt`. This will show what version of the prototype kit you’re using.
 
-3. Make a backup copy of your prototype folder.
+- If your prototype is version 8 or before, then contact the GOV.UK Prototype team for help with updating it
+- If your prototype is version 9 or 10, you can update to version 11 by following the steps on this page
 
-4. You’ll need to copy some files that operating systems do not usually show. These hidden files usually start with a `.` and are known as ‘dot files’. You can:
-    - [show hidden files in Windows](https://support.microsoft.com/en-us/windows/view-hidden-files-and-folders-in-windows-97fbc472-c603-9d90-91d0-1166d1d9f4b5)
-    - show hidden files on MacOS by pressing command + shift + . in Finder 
+### Updating to version 11
 
-5. In your prototype folder, delete everything except the `app` and `.git` folders.
+1. Make a backup copy of your prototype folder.
 
-6. Copy all the files and folders except the `app` folder from the `govuk-prototype-kit` folder to your prototype.
+2. Make a backup of your prototype folder. You can do this in Finder or Windows Explorer. This may take a few minutes. In the terminal, cd to your prototype folder.
```

Comment:
I think cd should be in backticks ` `` ` so that it is [formatted as code](https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code).

```suggestion
2. Make a backup of your prototype folder. You can do this in Finder or Windows Explorer. This may take a few minutes. In the terminal, `cd` to your prototype folder.
```